common/packet/include/packet.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef PACKET_H
+#define PACKET_H
+#include <stdio.h>
+#include <iostream>
```

Comment:
Do we really need these in Packet? I don't think anything's being printed

Reply:
You’re right - I deleted all the print statements.